pytest/readme.md

```diff
@@ -6,39 +6,54 @@ Simply run `exec_pytest.sh` (optinal flag `--verbose` and `--reset-submodules`)
 
 1. Ensure submodules are clean and point to the correct commit. Use the following commands at your own risk:
 
-```bash
-git submodule foreach --recursive git reset --hard
-git submodule foreach --recursive git clean -fdx
-git submodule update --init --recursive --force
-```
+    ```bash
+    git submodule foreach --recursive git reset --hard
+    git submodule foreach --recursive git clean -fdx
+    git submodule update --init --recursive --force
+    ```
 
 2. Build nearcore and main node:
 
-```bash
-# build nearcore:
-cd libs/nearcore && cargo build -p neard --release
+    ```bash
+    # build nearcore:
+    cd libs/nearcore && cargo build -p neard --release
 
-# build the main node
-cd ../.. && cargo build -p mpc-node --release --features=network-hardship-simulation
-```
+    # build the main node
+    cd ../.. && cargo build -p mpc-node --release --features=network-hardship-simulation
+    ```
 
 3. Set up virtualenv (optional, but recommended):
 
-```bash
-cd pytest && python3 -m venv venv
+    ```bash
+    cd pytest && python3 -m venv venv
 
-# activate virtualenv:
-source venv/bin/activate
+    # activate virtualenv:
+    source venv/bin/activate
 
-# install requirements:
-pip install -r requirements.txt
-```
+    # install requirements:
+    pip install -r requirements.txt
+    ```
 
-4. Run pytest:
+4. Install docker and cargo-near. For the latter:
 
-```bash
-pytest # -v -s optional flags for verbosity and -m "not slow" to skip slow tests
-```
+    ```bash
+    cargo install cargo-near
+    ```
+
+    This is only needed if using reproducible builds for the contract, which is 
+    enabled by default.
+
+5. Run pytest:
+
+    ```bash
+    pytest # -v -s optional flags for verbosity and -m "not slow" to skip slow tests
+    ```
+
+    To disable the reproducible build of the contract, use:
+
+    ```bash
+    pytest --non-reproducible
+    ```
```

Comment:
So, I know that to many people familiar with docker, this is something they will know. But it might still be useful to add something along the lines of  
```
In case you run into permission issues, make sure your user is part of the docker group, c.f. [docker docs](https://docs.docker.com/engine/install/linux-postinstall/)
```